specs/core/0_beacon-chain.md

```diff
@@ -502,10 +495,12 @@ The following data structures are defined as [SimpleSerialize (SSZ)](https://git
 {
```

Comment:
Should probably rename this `BeaconBlockHeader`

Reply:
Yes, done! 👍(As mentioned in the PR description I didn't rename to `BeaconBlockHeader` to keep the diff small.)